docs/sources/flow/reference/components/prometheus.exporter.kafka.md

```diff
@@ -0,0 +1,115 @@
+| `kafka_version`             | `string`        | Kafka broker version.                                                                                                                                                               | `2.0.0` | no       |
+| `use_zookeeper_lag`         | `bool`          | if you need to use a group from zookeeper.                                                                                                                                          |         | no       |
+| `zookeeper_uris`            | `array(string)` | Address array (hosts) of zookeeper server..                                                                                                                                         |         | no       |
+| `kafka_cluster_name`        | `string`        | Kafka cluster name.                                                                                                                                                                 |         | no       |
+| `metadata_refresh_interval` | `duration`      | Metadata refresh interval.                                                                                                                                                          | `1m`    | no       |
+| `allow_concurrency`         | `bool`          | If true, all scrapes will trigger kafka operations otherwise, they will share results. WARN: This should be disabled on large clusters.                                             | `true`  | no       |
+| `max_offsets`               | `int`           | Maximum number of offsets to store in the interpolation table for a partition.                                                                                                      | `1000`  | no       |
+| `prune_interval_seconds`    | `int`           | How frequently should the interpolation table be pruned, in seconds.                                                                                                                | `30`    | no       |
+| `topics_filter_regex`       | `string`        | Regex filter for topics to be monitored.                                                                                                                                            | `.*`    | no       |
+| `groups_filter_regex`       | `string`        | Regex filter for consumer groups to be monitored.                                                                                                                                   | `.*`    | no       |
+
+## Exported fields
+The following fields are exported and can be referenced by other components.
+
+| Name      | Type                | Description                                              |
+|-----------|---------------------|----------------------------------------------------------|
+| `targets` | `list(map(string))` | The targets that can be used to collect `kafka` metrics. |
```

Comment:
Why is Kafka in code here? And not just Kafka?

Reply:
Yes, It's only collect kafka metrics, So the word `kafka` is unnecessary here

Reply:
So should the text here be: 

> The targets that can be used to collect Kafka metrics.

(no code syntax, and capitalise Kafka)